source/gloperate/source/viewer/RenderSurface.cpp

```diff
@@ -20,7 +22,18 @@ RenderSurface::RenderSurface(ViewerContext * viewerContext)
 , m_frame(0)
 , m_mouseDevice(new MouseDevice(m_viewerContext->inputManager(), "Render Surface"))
 , m_keyboardDevice(new KeyboardDevice(m_viewerContext->inputManager(), "Render Surface"))
+, m_video(nullptr)
+, m_imageExporter(nullptr)
+, m_requestVideo(false)
 {
+    addFunction("createVideo", this, &RenderSurface::createVideo);
+    addFunction("exportImage", this, &RenderSurface::exportImage);
+
+    if (m_viewerContext->scriptEnvironment())
```

Comment:
Would be better to do that from the outside (e.g., ViewerContext in registerSurface?)